packages/block-editor/src/hooks/layout.js

```diff
@@ -172,7 +172,9 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 										layout: {
 											type:
 												layoutType?.name ===
-												'constrained'
+													'constrained' ||
```

Comment:
Would a `const legacyMarkup = !! inherit || !! contentSize` be helpful to clarify why this is needed - could also be used in the `checked` prop as well?

Reply:
Oh, that's much nicer! I'll update, and we can use it in the check for help text, too 👍

Reply:
Updated.